projects/batfish/src/test/java/org/batfish/common/BatfishTestUtils.java

```diff
@@ -0,0 +1,94 @@
+package org.batfish.common;
```

Comment:
This is in the `batfish` module, not the common module -- it should not be in the `org.batfish.common` package. I would suggest the same package as `Batfish` class.